docs/source/index.rst

```diff
@@ -64,8 +64,8 @@ designed and developed by network scientists with the needs of network scientist
 mind.
 
 - Repository: https://github.com/xgi-org/xgi
-- PyPI: https://pypi.org/project/xgi/
-- Documentation: https://xgi.readthedocs.io/
```

Comment:
Why remove the link to the docs?

Reply:
This might be a best practices thing, but I removed it because if the user is seeing the link (rendered as html), they will already be at the documentation homepage. Happy to add it back in.

Reply:
Okay gotcha, I guess that's fine.